samza-core/src/main/java/org/apache/samza/system/SystemAdmins.java

```diff
@@ -20,26 +20,30 @@
 package org.apache.samza.system;
 
 import java.util.Map;
+import java.util.Set;
 import org.apache.samza.SamzaException;
 import org.apache.samza.config.Config;
 import org.apache.samza.config.JavaSystemConfig;
+import org.apache.samza.config.MapConfig;
 
 
+/**
+ * Provides a mapping from system name to a {@link SystemAdmin}. Remember to start/stop this.
```

Comment:
Maybe: Needs to be started/stopped before/after use.

Reply:
Resolved